build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +56,22 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable the integrated Node IPAM controller within the Antrea controller.
+#enableNodeIPAMController: false
+
+# CIDR Ranges for Pods in cluster. Value can contain a single CIDR range, or multiple ranges, separated by commas.
+# The CIDRs could be either IPv4 or IPv6. Value ignored when enableNodeIPAMController is false.
+#clusterCIDRs:
+
+# CIDR Range for Services in cluster. It is not necessary to specify it when there is no overlap with clusterCIDRs.
+# Value ignored when enableNodeIPAMController is false.
+#serviceCIDR:
+
+# Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. Value ignored when enableNodeIPAMController is false
+# or when IPv4 Pod CIDR is not configured. Valid range is 16 to 30.
+#nodeCIDRMaskSizeIPv4:
+
+# Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. Value ignored when enableNodeIPAMController is false
+# or when IPv6 Pod CIDR is not configured. Valid range is 64 to 126.
+#nodeCIDRMaskSizeIPv6:
```

Comment:
there is no default value for `nodeCIDRMaskSizeIPv4` and `nodeCIDRMaskSizeIPv6`?